scripts/generateAssetData/utils/portals.ts

```diff
@@ -34,10 +34,8 @@ export const fetchPortalsTokens = async (
 
     const params = {
       limit: '250',
-      // Minimum 1000 bucks liquidity if asset is a LP token
-      minLiquidity: '1000',
-      // Minimum 1% APY if asset is a LP token
-      minApy: '1',
+      // Minimum 100,000 bucks liquidity if asset is a LP token
```

Comment:
Removed `minApy` as the LP-YCRV token this was originally spotted for has 0 APY, though has some liquidity on it still:

<img width="253" alt="image" src="https://github.com/user-attachments/assets/3377fbd4-1081-47f4-aceb-513e5f9538ab">
<img width="1244" alt="image" src="https://github.com/user-attachments/assets/723a1380-676a-433f-b17e-6e851fa16e39">

https://ycrv.yearn.fi/app/stake

Failure to do so won't fetch LP-YCRV (and others with 0 APY), however, Coingecko *will* fetch it so that wouldn't solve it for this token and similar ones.

Reply:
Curious but do you even understand why it has some liquidity with 0 apy? Mechanics doesn't permit to calculate any apy but is valid? as rFOX?

Reply:
Depends, could be
- some old pool (it's the case here) that's now inactive/deprecated, could e.g be set at smart-contract level despite still having liquidity
- APY not able to be calculated for any reason as you noted

But indeed definitely an odd, but valid scenario!